src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -176,10 +194,12 @@ CanonicalMostRead.propTypes = {
   endpoint: string.isRequired,
   constrainMaxWidth: bool.isRequired,
   maxTwoColumns: bool,
+  isOnFrontPage: bool,
 };
 
 CanonicalMostRead.defaultProps = {
   maxTwoColumns: false,
+  isOnFrontPage: false,
```

Comment:
It seems like each scenario needs a different outer wrapper, so maybe we should just export this with the generic wrapper and add the styles for each individual use case where it's used?

Reply:
My understanding of this is that we should do away with `maxTwoColumns`, `isOnFrontPage` and `constrainMaxWidth` props and just add the specific styling they add where the different props are set?

Reply:
> My understanding of this is that we should do away with maxTwoColumns, isOnFrontPage and constrainMaxWidth props and just add the specific styling they add where the different props are set. Is that right?

We can't get rid of `maxTwoColumns` because that's needed for the Most Read components. However with `isOnFrontPage`/`constraintMaxWidth`, these props are being used as wrappers to correctly position the feature on the specific page type. 

(constraintMaxWidth is currently for article pages)

Reply:
Hey, yeah the isOnFrontPage / constainMaxWidth is what I think should be added where the component is used :)